examples/feature_alignment_example/misalign_data.py

```diff
@@ -0,0 +1,40 @@
+import random
+from pathlib import Path
+from typing import List
+
+import pandas as pd
+
+
+def random_split_data(df: pd.DataFrame, n: int) -> List[pd.DataFrame]:
+    df_rand = df.sample(frac=1.0, random_state=42)
+    num_rows_per_df = len(df_rand) // n
+    smaller_dfs = [df_rand.iloc[i * num_rows_per_df : (i + 1) * num_rows_per_df] for i in range(n - 1)]
+    smaller_dfs.append(df_rand.iloc[(n - 1) * num_rows_per_df :])
+    return smaller_dfs
+
+
+if __name__ == "__main__":
+    data_path = Path("examples/feature_alignment_example/mimic3d.csv")
+    target_datapath = "examples/feature_alignment_example"
```

Comment:
If we're going to hardcode this destination, can we also direct it to the datasets folder? It would also be worthwhile to add these files to the gitignore file so that they don't accidentally get checked in.